exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Service.java

```diff
@@ -35,13 +34,23 @@
 
   /**
    * Configures the service instance. This method is called <em>once</em> after the service
```

Comment:
@alekseysidorov That's our take on the configuration documentation (+ Configuration.java)